universal-application-tool-0.0.1/test/repository/ProgramRepositoryTest.java

```diff
@@ -12,14 +9,14 @@
   public void createProgram() {
     final ProgramRepository repo = app.injector().instanceOf(ProgramRepository.class);
     Program program = new Program();
-    program.setName("program one");
-    program.setVersion(1L);
-    program.setObject(ImmutableMap.of("key", "value"));
-
-    repo.insertProgram(program).toCompletableFuture().join();
-
-    Program p = repo.lookupProgram("program one", 1L).toCompletableFuture().join().get();
-    assertThat(p.getVersion()).isEqualTo(1L);
-    assertThat(p.getObject()).containsEntry("key", "value");
+    //    program.setName("program one");
+    //    program.setVersion(1L);
+    //    program.setObject(ImmutableMap.of("key", "value"));
+    //
+    //    repo.insertProgram(program).toCompletableFuture().join();
+    //
+    //    Program p = repo.lookupProgram("program one", 1L).toCompletableFuture().join().get();
+    //    assertThat(p.getVersion()).isEqualTo(1L);
+    //    assertThat(p.getObject()).containsEntry("key", "value");
```

Comment:
suggestion (blocking): avoid checking in commented code.

i don't think we'll ever have a need for an async create action for programs, so this test and the repo code it tests can be safely removed.

Reply:
i kept the test. we can remove it in a separate PR if its really not needed.